src/Bank.java

```diff
@@ -0,0 +1,21 @@
+import javax.swing.text.StyledEditorKit;
```

Comment:
Ah yes.  Keep an eye on it when that happens so you don't include extra files for your program to load.